protocol/contracts/v2/safeGuards/ThresholdSafeGuard.sol

```diff
@@ -52,6 +52,12 @@ contract ThresholdSafeGuard is IThresholdSafeGuard, TempleElevatedAccess {
      */
     EnumerableSet.AddressSet internal _safeTxExecutors;
 
+    /**
+     * @notice Setting a signature threshold for ETH transfers is also possible by calling 
+     * `setFunctionThreshold(addr, `ETH_TRANSFER_SELECTOR`, signers)`
+     */
+    bytes4 public constant ETH_TRANSFER_SELECTOR = bytes4(keccak256("__ETH_TRANSFER__"));
```

Comment:
This was the incorrect fix, as it just shifts the same problem but to a different selector which may still clash.

This was fixed more appropriately in https://github.com/TempleDAO/temple/pull/842/commits/efb261e19e672cae71c4093b6b941cd833f066d5